cmake/defaults/Options.cmake

```diff
@@ -72,7 +72,7 @@ option(PXR_ENABLE_GL_SUPPORT "Enable OpenGL based components" ON)
 
 # Precompiled headers are a win on Windows, not on gcc.
```

Comment:
can we also revise the comment to not say Windows, but rather "msvc with msbuild"

Reply:
Maybe just remove this comment - or do the homework and measure this properly :-)

Reply:
On another note - I've been using sccache with windows msvc USD builds recently, and been pretty happy with it so far.  Haven't done any time comparisons, but rebuilds feel faster - particularly if you switch branches and come back, so a lot of stuff has updated timestamps but hasn't actually changed.